src/test/java/pages/ChangePasswordPage.java

```diff
@@ -0,0 +1,52 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ChangePasswordPage {
+
+    String fulHeaderButtonId = "//header/nav/div/div[3]/ul/li[4]/a/span/span/img";
```

Comment:
Please, remove \' and \" 
Correct: "//*[@id='change-password-submit']"
Wrong: "//*[@id=\'change-password-submit\']"
Wrong: "//*[@id=\"change-password-submit\"]"